library/std/src/os/mod.rs

```diff
@@ -141,11 +141,12 @@ pub mod openbsd;
 pub mod redox;
 #[cfg(target_os = "solaris")]
 pub mod solaris;
-
 #[cfg(target_os = "solid_asp3")]
 pub mod solid;
 #[cfg(target_os = "vxworks")]
 pub mod vxworks;
+#[cfg(target_os = "watchos")]
+pub mod ios;
```

Comment:
Ah, can you hold off on this until another PR? Is this required to compile?

Reply:
I think it is required - what is the issue?

Reply:
It needs libs-api decision, I think -- the outcome of the libs meeting was just to pull this part out into another PR and land it separately if possible, so that more deliberation can happen.

I definitely don't think it should be called `ios`, and there are a lot of reasons the two might diverge (they're not even quite the same architecture, for example).

Anyway, please submit a separate PR with this part (or what you had before) and we can land the rest of this, and work out what to do with the `std::os` bit.

Reply:
ok removed those parts